client/src/components/utils/anchor.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Icon } from "antd";
+
+export default props =>
+  props.socialLinks.map(href => (
```

Comment:
@aminalakhsham I would suggest either changing the name of this component to be more specific (like "SocialLinksList") or changing the prop name to be more generic (like just "links") since right now the component name is pretty generic but the prop that it requires is pretty specific.